src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/util/HomeOfficeApiUtil.java

```diff
@@ -6,18 +6,32 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.collect.ImmutableMap;
+import java.time.format.DateTimeFormatter;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.UUID;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.MediaType;
 import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.DateProvider;
 import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.AsylumCase;
 import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.AsylumCaseDefinition;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.SystemDateProvider;
 import uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.client.HomeOfficeResponseException;
 
 @Service
 public class HomeOfficeApiUtil {
```

Comment:
Let's make this specific to 1 purpose rather than calling it Util class.  You can split it into multiple objects too.

Reply:
It has only 3 methods helping create request. It is already renamed and fixed in #77

Reply:
Please can you link to the specific change in the PR you are referring to?  I don't think this is particularly complicated so ought to be addressed in this PR.

Reply:
Renamed to HomeOfficeRequestHelper